### DISCUSSION TITLE
Signal processing from mediamtx server

### DISCUSSION THREAD

**Discussion by AxelHamburch** [ossrs/WordPress-Plugin-SrsPlayer#6] [General]
I use the [https://github.com/bluenviron/mediamtx](https://github.com/bluenviron/mediamtx) server and create a video stream in WebRTC format like this: 

http://localhost:8889/mystream (like http://1.2.3.4:8889/mystream or with https over caddy reverse proxy)

See the corresponding documentation page:
[https://github.com/bluenviron/mediamtx#webrtc](https://github.com/bluenviron/mediamtx#webrtc)

If I run this stream in the browser, it works perfectly. But I can't find a WordPress player that can play it. Can the SrsPlayer perhaps do that, or in which format would I have to format it for the SrsPlayer so that it works? Any ideas?

Cheers, Axel

**Comment by jayfan0** [comment]
Same problem here. I can't get WebRTC, HLS or any other protocol to work. Did you end up finding a solution?

**Reply by AxelHamburch** [reply]
Yes, I have found a way. I operate a small ESP32 mini camera. My MediaMTX server picks up the rtsp stream. I add music and a QR code image to the stream with ffmpeg and MediaMTX then makes the stream available under a subdomain of mine.

You can access the stream here under Example 3 / Webcam.:
https://ereignishorizont.xyz/onboarding/en/#Example_3_-_Live-Peepshow

A small window opens in which the stream is displayed. Since it is a small mini camera and I wanted to keep the data rate low, the resolution is low, but it works for illustration purposes. 

This is the HTML code for the call from my website link:

`<p align="center"><br><br><br><a href="https://rtsp.ereignishorizont.xyz/mystream/" onclick="window.open(this.href, 'window', 'width=640,height=480'); return false"> 👉 Click here to go to the <big> Webcam </big> 👈 </a><br>Have 🙏 a little patience</p>`

The Caddy web server reverse proxy linking is quite simple:

rtsp.ereignishorizont.xyz {
    reverse_proxy localhost:8889
}

**Reply by jayfan0** [reply]
Thanks. I found this option too - to use a reverse proxy for the stream and embed it with an iframe. The plugin didnt work at all but i suppose less plugins on wordpress is always a good thing!
